inference-engine/tests/functional/plugin/cpu/single_layer_tests/eltwise.cpp

```diff
@@ -333,5 +333,33 @@ const auto params_5D_Planar_Blocked = ::testing::Combine(
 
 INSTANTIATE_TEST_CASE_P(smoke_CompareWithRefs_5D_Planar_Blocked, EltwiseLayerCPUTest, params_5D_Planar_Blocked, EltwiseLayerCPUTest::getTestCaseName);
 
+
+std::vector<std::vector<std::vector<size_t>>> inShapes_4D_1D = {
```

Comment:
I added `5D_1D` as a common shape. I think there is no reason to add symmetric cases, since changing the input numbers does not change the execution flow.